src/js/components/pageTitle.js

```diff
@@ -0,0 +1,18 @@
+const pageTitle = data => {
+  const section = document.createElement('section');
+  const eyebrow = document.createElement('p');
+  const heading = document.createElement('h1');
```

Comment:
Tag names should be uppercase.

Reply:
There is no difference between uppercase and lowercase tag names in the `createElement` method. As a way to keep consistency between the code, you can use the tag names here in lowercase.